### ISSUE TITLE
train_text_to_video_sft 中的设置

### ISSUE THREAD

**Issue by lijain** [a-r-r-o-w/finetrainers#94]
你好，请问下# Single GPU 训练时
train_text_to_video_sft.sh
ACCELERATE_CONFIG_FILE="accelerate_configs/uncompiled_1.yaml"
训练没有问题。我将uncompiled_1.yaml-->deepspeed.yaml换为deepspeed报错，
![企业微信截图_17325422283473](https://github.com/user-attachments/assets/754d9d0d-9684-499c-a611-c7b0de7e7927)
请问下使用deepspeed该怎么改

**Comment by sayakpaul** [comment]
Can we move to English?

**Comment by zy0406** [comment]
cogvideox_image_to_video_lora.py with 1.5 I2V model

The current code does not support training for version 1.5,  right?

**Comment by sayakpaul** [comment]
We know deepseepd is supported. Without a full handle on what is your accelerate config and error trace we cannot do much. 

https://github.com/a-r-r-o-w/cogvideox-factory/pull/10#issuecomment-2400314869

**Comment by lijain** [comment]
Yeah, I thought it was weird, too. My Settings are as follows
![企业微信截图_1732621221375](https://github.com/user-attachments/assets/7e15011d-9ad5-42b8-9c12-02e573d108fe)

**Comment by sayakpaul** [comment]
Can you paste these settings instead of screenshots?

**Comment by lijain** [comment]
> We know deepseepd is supported. Without a full handle on what is your accelerate config and error trace we cannot do much.
> 
> [#10 (comment)](https://github.com/a-r-r-o-w/cogvideox-factory/pull/10#issuecomment-2400314869)
I also read your Settings and mine are basically similar, I use A100 80g should not have a memory deficit
deepspeed.yaml:
compute_environment: LOCAL_MACHINE
debug: false
deepspeed_config:
  gradient_accumulation_steps: 1
  gradient_clipping: 1.0
  offload_optimizer_device: cpu
  offload_param_device: cpu
  zero3_init_flag: false
  zero_stage: 2
distributed_type: DEEPSPEED
downcast_bf16: 'no'
#enable_cpu_affinity: false
machine_rank: 0
main_training_function: main
mixed_precision: bf16
num_machines: 1
num_processes: 1
rdzv_backend: static
same_network: true
tpu_env: []
tpu_use_cluster: false
tpu_use_sudo: false
use_cpu: false

sh：
export TORCH_LOGS="+dynamo,recompiles,graph_breaks"
export TORCHDYNAMO_VERBOSE=1
export WANDB_MODE="offline"
export NCCL_P2P_DISABLE=1
export TORCH_NCCL_ENABLE_MONITORING=0

GPU_IDS="0"

# Training Configurations
# Experiment with as many hyperparameters as you want!
LEARNING_RATES="1e-4"
LR_SCHEDULES="cosine_with_restarts"
OPTIMIZERS="adamw"
MAX_TRAIN_STEPS="20000"

# Single GPU uncompiled training
ACCELERATE_CONFIG_FILE="accelerate_configs/deepspeed.yaml"

# Absolute path to where the data is located. Make sure to have read the README for how to prepare data.
# This example assumes you downloaded an already prepared dataset from HF CLI as follows:
#   huggingface-cli download --repo-type dataset Wild-Heart/Tom-and-Jerry-VideoGeneration-Dataset --local-dir /path/to/my/datasets/tom-and-jerry-dataset
DATA_ROOT="/dataset/gen_gif/Dance-VideoGeneration-Dataset"
CAPTION_COLUMN="captions.txt"
VIDEO_COLUMN="videos.txt"
output_dir="/nProject/cogvideox-factory/log/cogvideox_sft_optimizer_${optimizer}_steps_${steps}_lr-schedule_${lr_schedule}__lr_${learning_rate}/"
# Launch experiments with different hyperparameters
accelerate launch --config_file $ACCELERATE_CONFIG_FILE --gpu_ids $GPU_IDS training/cogvideox_image_to_video_sft2.py \
      --pretrained_model_name_or_path /nProject/zpretrain_ckpt/CogVideoX1.5-5B-I2V \
      --data_root $DATA_ROOT \
      --caption_column $CAPTION_COLUMN \
      --video_column $VIDEO_COLUMN \
      --height_buckets 480 \
      --width_buckets 720 \
      --frame_buckets 53 \
      --dataloader_num_workers 8 \
      --pin_memory \
      --num_validation_videos 1 \
      --validation_epochs 1 \
      --seed 42 \
      --mixed_precision bf16 \
      --output_dir $output_dir \
      --max_num_frames 53 \
      --train_batch_size 1 \
      --max_train_steps $MAX_TRAIN_STEPS \
      --checkpointing_steps 2000 \
      --gradient_accumulation_steps 1 \
      --gradient_checkpointing \
      --learning_rate $LEARNING_RATES \
      --lr_scheduler $LR_SCHEDULES \
      --lr_warmup_steps 800 \
      --lr_num_cycles 1 \
      --enable_slicing \
      --enable_tiling \
      --optimizer $OPTIMIZERS \
      --beta1 0.9 \
      --beta2 0.95 \
      --weight_decay 0.001 \
      --max_grad_norm 1.0 \
      --allow_tf32 \
      --nccl_timeout 1800

**Comment by sayakpaul** [comment]
Can you format the commands?
